### PR TITLE
Updating all examples

### PR DESCRIPTION
**Pull request by romfrancois**
#### Add example checklist

- [x] Updating examples with their dependencies

### CODE REVIEW

File: angular/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # TradingView Charting Library and Angular 5 Integration Example
```
Comment (illetid) [inline angular/README.md:1]: I think we need to update version here or remove version form the title

Comment (illetid) [reply angular/README.md:1]: I mean angular 5, from `package.json` looks like there is 18 already

File: nuxtjs/package.json [hunk 2]
```diff
@@ -9,10 +9,10 @@
...
   },
   "dependencies": {
-    "core-js": "~3.27.2",
-    "nuxt": "~2.15.8"
+    "core-js": "~3.37.1",
+    "nuxt": "~2.18.1"
   },
   "devDependencies": {
-	"vue-server-renderer": "2.7.14"
+	  "vue-server-renderer": "2.7.16"
```
Comment (illetid) [inline nuxtjs/package.json:16]: [minor] not sure if this space is needed here

**Review by illetid — APPROVED** [review]
LGTM, few minor comment that could be ignored

**Comment by romfrancois** [comment]
> LGTM, few minor comment that could be ignored

Have pushed another commit to fix those.

**Review by illetid — APPROVED** [review]

File: nextjs/package.json [hunk 3]
```diff
@@ -8,12 +8,12 @@
...
   },
   "dependencies": {
-    "@types/node": "18.15.10",
-    "@types/react": "18.0.30",
-    "@types/react-dom": "18.0.11",
-    "next": "13.2.4",
-    "react": "18.2.0",
-    "react-dom": "18.2.0",
-    "typescript": "5.0.2"
+    "@types/node": "^22.0.0",
```
Comment (SlicedSilver) [inline nextjs/package.json:11]: Maybe if we are setting the version of the types to v22 then we should enforce a node version in the package.json as well:
```
"engines": {
    "node": ">=22.0.0"
  },
```

**Review by SlicedSilver — APPROVED** [review]

File: nextjs/package.json (hunk 3 above)
Comment (romfrancois) [reply nextjs/package.json:14]: Done

**Pull request merged**